package.json

```diff
@@ -31,6 +31,7 @@
     "ember-ajax": "2.0.1",
     "ember-autosave": "2.0.1",
     "ember-bootstrap": "0.7.2",
+    "ember-browserify": "^1.1.13",
```

Comment:
So this is actually a little different. You should move ember-browserify to the dependencies section (instead of devDependencies) of ember-osf. That way it *should* get installed when your run yarn and don't have to explicitly include it in consuming apps that rely on ember-osf, unless they actually need it in that the consuming app itself, which in this case we don't.